content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -93,6 +93,371 @@ service:
+
+##### Receive Traces From Host
+
+- Create a `otel_collector_config.yaml` file. [Here is an example template](https://docs.datadoghq.com/tracing/setup_overview/open_standards/#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's otlp receiver and datadog exporter.
+
+- Use a published docker image such as [`otel/opentelemetry-collector-contrib:latest`](https://hub.docker.com/r/otel/opentelemetry-collector-contrib/tags)
+
+- OpenTelemetry Traces are sent to the OpenTelemetry Collector over TCP or UDP on a number of ports, which must be exposed on the container.  By default traces are sent over `OTLP/gRPC on port 55680`, but common protocols and their ports include: 
+
+  ```
+  Zipkin/HTTP on port 9411
+  Jaeger/gRPC on port 14250
+  Jaeger/HTTP on port 14268
+  Jaeger/Compact on port 6831 (UDP)
+  OTLP/gRPC on port 55680
+  OTLP/HTTP on port 55681
+  ```
+
+- Run the container with the configured ports and an `otel_collector_config.yaml` file. For example:
+
+  ```
+   $ docker run \
+    -p 55680:55680 \
+    -v $(pwd)/otel_collector_config.yaml:/etc/otel/config.yaml \
+    otel/opentelemetry-collector-contrib:latest
+  ```
+  
+- Configure your application with the appropriate Resource attributes for unified service tagging by adding the metadata [described here](https://docs.datadoghq.com/tracing/setup_overview/open_standards/#opentelemetry-collector-datadog-exporter)
+
+##### Receive Traces From Other Containers
+
+- Create a `otel_collector_config.yaml` file. [Here is an example template](https://docs.datadoghq.com/tracing/setup_overview/open_standards/#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's otlp receiver and datadog exporter.
+
+
+- Configure your application with the appropriate Resource attributes for unified service tagging by adding the metadata [described here](https://docs.datadoghq.com/tracing/setup_overview/open_standards/#opentelemetry-collector-datadog-exporter)
+
+- Create a docker network 
+
+  `docker network create <NETWORK_NAME>`
+  
+- Run the OpenTelemetry Collector container and Application container in the same network. *Note*: When running the application container, ensure the environment variable `OTEL_EXPORTER_OTLP_SPAN_ENDPOINT` is configured to use the appropriate hostname for the Collector. In the example below this would be `opentelemetry-collector`
+ 
+  ```
+  # Datadog Agent
+  docker run -d --name opentelemetry-collector \
+            --network <NETWORK_NAME> \
+            -v /var/run/docker.sock:/var/run/docker.sock:ro \
+            -v /proc/:/host/proc/:ro \
+            -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+            -v $(pwd)/otel_collector_config.yaml:/etc/otel/config.yaml \
+            otel/opentelemetry-collector-contrib:latest
```

Comment:
If we keep this and we don't rely on the official docs we can remove the tag here since it will pick `latest` by default
```suggestion
            otel/opentelemetry-collector-contrib
```